File: front/src/app/models/creditoscarbono.ts
import { Empresa } from './empresa';

export class Creditoscarbono {
    id?: number;
    idEmpresa?: number | null;
    quantidadeCreditos?: number | null;
    dataTransacao?: Date | null;
    tipoTransacao?: string | null;
    idEmpresaNavigation?: Empresa | null;
}

File: back/FilmesApi/FilmesApi/Models/CreditosCarbono.cs
using System;
using System.Collections.Generic;

namespace FilmesApi.Models;

public partial class Creditoscarbono
{
    public int Id { get; set; }

    public int? IdEmpresa { get; set; }

    public int? QuantidadeCreditos { get; set; }

    public DateTime? DataTransacao { get; set; }

    public string? TipoTransacao { get; set; }

    public virtual Empresa? IdEmpresaNavigation { get; set; }
}
